bin/kubectl-forward

```diff
@@ -8,7 +8,7 @@ describe() {
 help() {
   echo -e "
 Usage:
-  forward [-n namespace] host [sport:]dport
+  forward [--context context] [-n namespace] host [sport:]dport
```

Comment:
Let's keep this code consistent and use short options. I would suggest `-c context` here.

Reply:
Just wanted to be in line with other `kubectl` commands here. There's just `--context` available.

Reply:
I understand, nevertheless using short options is cleaner here imho. We already do it for the `namespace` which is also supported natively by the `kubectl`. So, Iwould either: 
* change the `context` to just `c` to stay consistent with the current style, 
* or remove it completely in order to allow users to pass any `kubectl` built-in options, but without creating an explicits support for them here. In this scenario I would leave a note in readme to let users know they could find these options with the `kubectl options`.

Reply:
I see. But it's a bit different with `--namespace` as `kubectl` also supports `-n` as alias for it while `-c` isn't an alias for `--context` but is used to specify a config file.

Reply:
Sure, no worries, there is no rush, feel free to come back to it when you have more time.

Reply:
I'll keep using my fork for now.